tox.ini

```diff
@@ -16,7 +16,7 @@ commands =
     black --check src
     isort --check src
     flake8 --config={toxinidir}/tox.ini src
-    pytest
+    pytest --cov
```

Comment:
why move cov here instead of in the toml?

Reply:
reverted